task/kind/0.1/README.md

```diff
@@ -0,0 +1,26 @@
+# kind
+
+This configures a kind (Kubernetes-in-Docker) environment to be ran in a Task.
+
+See https://kind.sigs.k8s.io for more details.
+
+### Workspaces
+
+- **source**: A workspace that contains the working directory to share among
+  Tasks.
+
+### Parameters
+
+| Name    | Description                                                                                                             |
+| ------- | ----------------------------------------------------------------------------------------------------------------------- |
+| image   | kind runtime image to use. Users should override this if they require particular CLI tools as part of their kind tests. |
+| command | command to run                                                                                                          |
+
+### Usage
+
+This task sets up settings needed to run `kind` in a Tekton Task using
+Docker-in-Docker. Because of this, the Task runs in privileged mode.
+
+Within the task, users can create and interact with the `kind` clusters like
+normal. See https://github.com/wlynch/tekton-kind for a simple example for how
```

Comment:
NIT: It would be nice to have an example of usage in the README, or at least a link to the example in the test folder.

Reply:
Done.